uiTests/src/test/scala/com/virtuslab/gitmachete/uitest/OpenTabActionUiTest.scala

```diff
@@ -12,11 +11,21 @@ import com.virtuslab.ideprobe.{ IntegrationTestSuite, IntelliJFixture }
 import org.junit.{ Assert, Test }
 
 class OpenTabActionUiTest extends BaseGitRepositoryBackedTest with IntegrationTestSuite {
-  val currentDir: File = new File(System.getProperty("user.dir"))
-  val rootDir: File = currentDir.getParentFile
-  val distDir: File = rootDir.toPath.resolve("build").resolve("distributions").toFile
-  val latestFileInDistDir: File = distDir.listFiles().maxBy(_.lastModified)
-  val pluginUri: URI = new URI(latestFileInDistDir.getAbsolutePath)
+
+  implicit class ProjectDirOps(projectDir: File) {
+
+    /** @return the latest file in <projectDir>/build/distributions */
+    def pluginFile: File = projectDir.toPath
```

Comment:
Scala's `implicit class`: in a add a method called `pluginFile` to the **existing** class, `java.io.File` specifically here.

Reply:
Update: https://projectlombok.org/features/experimental/ExtensionMethod for Java

Reply:
...but still not supported by IntelliJ Lombok plugin (https://github.com/mplushnikov/lombok-intellij-plugin/issues/21)